helm/charts/domain-rrd/templates/deployment.yaml

```diff
@@ -41,23 +41,3 @@ spec:
             {{- include "vro.redisClient.envVars" . | nindent 12 }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
```

Comment:
Why are helm files changing in this PR or is this an unresolved conflict?

Reply:
I think that change is come from develop branch. I just merge develop into my branch again this morning.